### PR TITLE
new topic on voting

### PR DESCRIPTION
**Pull request by cellio**
Voting help, needed for abilities

### CODE REVIEW

File: User-Help/Voting.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+| URL slug | Category |
+|:--------:|:--------:|
+| voting | Guidance |
+
+# Why should I vote?
+
+Voting is the primary indicator of valuable content.  Vote up posts that add to the community, and vote down ones that detract from it.  (Many posts do neither; you don't have to vote on everything.)
+
+Voting influences a post's score.  Post score contributes to earning abilities.  When you vote, you are, in a sense, saying "I would like the community to get more (or less) of this kind of post".
```
Comment (sarahmccuan) [inline User-Help/Voting.md:9]: When you vote, you are in a sense saying, "I would like the community..." 

More natural comma placement?

File: User-Help/Voting.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+
+# When should I vote?
+
+![voting buttons](/assets/voting.png]
+
+You can vote up or down on posts by clicking the arrows next to the post.  In general, vote up content that is clear, interesting, well-supported, and, for answers, correct.  Vote down content that is unclear or demonstrably wrong.
```
Comment (sarahmccuan) [inline User-Help/Voting.md:16]: Remove the "In general" from the second sentence?

File: User-Help/Voting.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+**Voting is not a popularity contest.**  Don't vote based on who posted something -- don't vote for your friends or vote down posts from people you don't like.
```
Comment (sarahmccuan) [inline User-Help/Voting.md:18]: We should discuss em vs en dashes, double en is probably not ideal?

File: User-Help/Voting.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+There is a daily limit on the number of votes you can cast.  This limit can be changed per community; the default is 30.
+
+
+# When should I change my vote?
+
+You can remove or reverse your vote at any time.  If you downvoted and the author edits to address your objection, you should at least remove your downvote (and upvote if you now find it appropriate).  If you've reconsidered -- maybe you have more information now than when you first voted -- then you can change your vote.  You aren't locked in; if you no longer agree with one of your past votes, change it.
+
+
+# What else can I do?
```
Comment (sarahmccuan) [inline User-Help/Voting.md:35]: imo this is one of the most important and most neglected pieces for downvoting context. Can we move this section higher or highlight it somehow?

**Review by sarahmccuan — APPROVED** [review]
Looks good to me!

File: User-Help/Voting.md (hunk 3 above)
Comment (cellio) [reply User-Help/Voting.md]: Some of them should have been other punctuation (fixed).  I replaced the remaining ones with &emdash;, which I think works in Markdown?

File: User-Help/Voting.md (hunk 4 above)
Comment (cellio) [reply User-Help/Voting.md]: Moved up.

**Review by sarahmccuan — APPROVED** [review]

**Pull request merged**